cobigen-cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -385,7 +384,7 @@ private void setOutputRootPath(File inputProject) {
                         : ((TemplateTo) matching.get(index)).getId();
                     logger.info("(" + selectedArtifactNumber + ") " + artifactDescription);
                 } catch (IndexOutOfBoundsException e) {
-                    logger.error("The " + artifactType + " number you have specified is out of bounds!");
+                    logger.error("The " + artifactType + " number you have specified is out of bounds ,So please provide the number(s) of increments that you want to generate separated by comma.!");
```

Comment:
```Java
So please provide the number(s) of increments that you want to generate separated by comma
```
That message doesn't make sense to me. This exception is thrown when the user selects an increment number that doesn't exist (for instance `459`, we only have 15 increments at most). Therefore the message you have added is not needed